### ISSUE TITLE
Settings: No option to select OpenPGP provider

### ISSUE THREAD

**Issue by glowinthedark** [android-password-store/Android-Password-Store#556]
In the Settings dialog there is no option anymore to select the OpenPGP Provider.

Version: latest master branch (built locally on 2019-oct-20): c9b550e[](https://github.com/android-password-store/Android-Password-Store/commit/c9b550e59c5f79515c38da44ec0c470824085e62) 

The only option available under the **Crypto** section is **Select OpenPGP Key Id**.

**Comment by msfjarvis** [comment]
I'm aware. It's blocked by OpenKeychain's archaic library that uses deprecated `android.support` libraries. Please start an issue at [their repo](https://github.com/open-keychain/openpgp-api) asking for them to support AndroidX so that modern and maintained apps can continue using their app.

**Comment by glowinthedark** [comment]
Thanks for the prompt reply! Will follow up with OpenKeychain then.

**Comment by glowinthedark** [comment]
@msfjarvis: apparently the [openpgp-api](https://github.com/open-keychain/openpgp-api) project maintainers have disabled issue tracking for the repo. [Your fork](https://techgaun.github.io/active-forks/index.html#https://github.com/open-keychain/openpgp-api) seems to be the most active.

**Comment by msfjarvis** [comment]
> @msfjarvis: apparently the [openpgp-api](https://github.com/open-keychain/openpgp-api) project maintainers have disabled issue tracking for the repo. [Your fork](https://techgaun.github.io/active-forks/index.html#https://github.com/open-keychain/openpgp-api) seems to be the most active.

Try the main repository. I created a [issue](https://github.com/open-keychain/open-keychain/issues/2432) before but it died down and the maintainers seem uninterested.

**Comment by glowinthedark** [comment]
I don't think there is much point to create a duplicate when another issue is already open. I upvoted it and hopefully enough people will upvote the issue so that the maintainers are finally motivated to take it up.

**Comment by msfjarvis** [comment]
Fixed on master
